scout/constants/indexes.py

```diff
@@ -123,8 +123,8 @@
         IndexModel([("synopsis", TEXT)], default_language="english", name="synopsis_text"),
         IndexModel([("causatives", ASCENDING)], name="causatives"),
         IndexModel(
-            [("collaborators", ASCENDING), ("updated_at", ASCENDING)],
-            name="collaborators_updated_at",
+            [("collaborators", ASCENDING), ("status", ASCENDING), ("updated_at", ASCENDING)],
```

Comment:
This speeds up the other query (nr_cases_by_status), no? Nice!

Reply:
Exactly!

Reply:
What is left to speed up if needed later is the third query, from `cases()` that actually cursors the documents.